### PR TITLE
fix: stop overwriting chain-id in testnode

### PR DESCRIPTION
**Pull request by Frenzoid**
## Overview

Closes https://github.com/celestiaorg/celestia-app/issues/2026

Now the testnode uses the configured chain-id and relies on the default to set the initial random value instead of overwriting it with a random value.

## Checklist

- [X] New and updated code has appropriate documentation
- [X] New and updated code has new and/or updated testing
- [X] Required CI checks are passing
- [X] Visual proof for any user facing features like CLI or documentation updates
- [X] Linked issues closed with keywords

### CODE REVIEW

File: test/util/testnode/full_node.go
Comment (evan-forbes) [inline test/util/testnode/full_node.go:174]: this was a good idea @Frenzoid, but using a string of "" is valid, so we can just leave it up to the users on this one

all we have to do is blindly use the chain-id in the config

**Review by evan-forbes — COMMENTED** [review]
thanks for handling this @Frenzoid !

**Comment by evan-forbes** [comment]
also, don't worry about the docker build CI failing, it will for all PRs from forks. we should expect the rest to pass though before merging

**Comment by rootulp** [comment]
@Frenzoid this repo attempts to conform to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) so I renamed the PR title b/c the PR title will be used as the default commit message for the squashed commit that ends up on main.

**Comment by Frenzoid** [comment]
Thank you all for the feeback, tbh its the first time i've seen this repo, and i've reached to it from LW3 :)

File: test/util/testnode/full_node.go
Comment (Frenzoid) [reply test/util/testnode/full_node.go:174]: I see, that make sense, on it right now :)

**Review by rootulp — CHANGES_REQUESTED** [review]
@Frenzoid golangci-lint is complaining b/c this import is no longer used:

```go
	tmrand "github.com/tendermint/tendermint/libs/rand"
```

I can't comment directly on the line but it should be removed

**Review by evan-forbes — APPROVED** [review]
approving pending CI :slightly_smiling_face: 

thanks again @Frenzoid

**Review by rootulp — APPROVED** [review]

**Review by cmwaters — APPROVED** [review]

**Pull request merged**
